pytket/binders/circuit/Circuit/add_op.cpp

```diff
@@ -466,6 +466,19 @@ void init_circuit_add_op(py::class_<Circuit, std::shared_ptr<Circuit>> &c) {
           "qubits: The qubits to append the box to"
           "\n:return: the new :py:class:`Circuit`",
           py::arg("pauliexpbox"), py::arg("qubits"))
+      .def(
+          "add_toffolibox",
+          [](Circuit *circ, const ToffoliBox &box, const qubit_vector_t &qubits,
+             const py::kwargs &kwargs) {
+            return add_box_method<UnitID>(
+                circ, std::make_shared<ToffoliBox>(box),
+                {qubits.begin(), qubits.end()}, kwargs);
+          },
+          "Append a :py:class:`ToffoliBox` to the "
```

Comment:
Can the line breaks be improved here, just for readability?

Reply:
As all Box operations in this file have similarly poorly formatted line breaks, suggestion is to update all of them in a separate PR.